service/awsconfig/v5/templates/cloudformation/guest/instance.go

```diff
@@ -18,6 +18,8 @@ const Instance = `{{define "instance"}}
         Value: {{ .ClusterID }}-master
   EtcdVolume:
     Type: AWS::EC2::Volume
+    DependsOn:
+      - MasterInstance
```

Comment:
I am not a YAML expert but I usually see lists without indentation. I think it should be this. 

```
+    DependsOn:
+    - MasterInstance
```

Reply:
Good point, done.